pkg/config/images.go

```diff
@@ -1,12 +1,24 @@
 package config
 
 import (
+	"fmt"
 	"os"
 	"strings"
 
 	acsapi "github.com/openshift/openshift-azure/pkg/api"
 )
 
+// openshiftVersion converts a VM image version (e.g. 310.14.20180101) to an
+// openshift container image version (e.g. v3.10.14)
+func openShiftVersion(imageVersion string) (string, error) {
+	parts := strings.Split(imageVersion, ".")
```

Comment:
I understand how you wanted to use split and use the returned string for `part[0]` and use array index for the version.